Modules/FIT/FDD/src/PostProcTask.cxx

```diff
@@ -185,6 +185,46 @@ void PostProcTask::initialize(Trigger, framework::ServiceRegistryRef services)
       getObjectsManager()->startPublishing(pairHistBC.first->second);
     }
   }
+  const auto& lut = o2::fdd::SingleLUT::Instance().getVecMetadataFEE();
+  auto lutSorted = lut;
+  std::sort(lutSorted.begin(), lutSorted.end(), [](const auto& first, const auto& second) { return first.mModuleName < second.mModuleName; });
+  uint8_t binPos{ 0 };
+  for (const auto& lutEntry : lutSorted) {
+    const auto& moduleName = lutEntry.mModuleName;
+    const auto& moduleType = lutEntry.mModuleType;
+    const auto& strChID = lutEntry.mChannelID;
+    const auto& pairIt = mMapFEE2hash.insert({ moduleName, binPos });
+    if (pairIt.second) {
+      binPos++;
+    }
+    if (std::regex_match(strChID, std::regex("[[\\d]{1,3}"))) {
+      int chID = std::stoi(strChID);
+      if (chID < sNCHANNELS_PM) {
+        mChID2PMhash[chID] = mMapFEE2hash[moduleName];
+      } else {
+        ILOG(Error, Support) << "Incorrect LUT entry: chID " << strChID << " | " << moduleName << ENDM;
+      }
+    } else if (moduleType != "TCM") {
+      ILOG(Error, Support) << "Non-TCM module w/o numerical chID: chID " << strChID << " | " << moduleName << ENDM;
+    } else if (moduleType == "TCM") {
+      uint8_t mTCMhash = mMapFEE2hash[moduleName];
+    }
+  }
```

Comment:
Can you please move this functionality as helper function like here https://github.com/AliceO2Group/QualityControl/blob/3584e9b886b8ef16166336f11f298c976ba4df45/Modules/FIT/Common/include/FITCommon/HelperLUT.h#L26 into the common section? We have a dozens of this piece of code...